packages/fether-react/src/Send/TxForm/TxDetails/TxDetails.js

```diff
@@ -0,0 +1,104 @@
+// Copyright 2015-2018 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+import React, { Component } from 'react';
+import BigNumber from 'bignumber.js';
+import { fromWei, toWei } from '@parity/api/lib/util/wei';
+import { removeTrailingZeros } from '../../../utils/removeTrailingZeros';
+
+class TxDetails extends Component {
+  state = {
+    showDetails: false
+  };
+
+  renderCalculation = () => {
+    const { estimatedTxFee, values } = this.props;
+
+    const gasPriceBn = new BigNumber(values.gasPrice.toString());
+    const gasLimitBn = estimatedTxFee
+      .div(gasPriceBn)
+      .div(10 ** 9)
+      .toFixed(0)
+      .toString();
+
+    return `Estimate amount of gas: ${gasLimitBn}`;
+  };
+
+  renderDetails = () => {
+    return `${this.renderCalculation()}
+${this.renderFee()}
+${this.renderTotalAmount()}`;
+  };
+
+  renderFee = () => {
+    const { estimatedTxFee } = this.props;
+
+    return `Fee: ${fromWei(estimatedTxFee, 'ether')
+      .toFixed(9)
+      .toString()} ETH (estimate * gas price)`;
+  };
+
+  renderTotalAmount = () => {
+    const { estimatedTxFee, token, values } = this.props;
+
+    return `Total Amount: ${removeTrailingZeros(
+      fromWei(
+        estimatedTxFee.plus(
+          token.address === 'ETH' ? toWei(values.amount.toString()) : 0
+        ),
+        'ether'
+      )
+        .toFixed(18)
```

Comment:
If you remove this line, then `.toString()` actually already removes trailing zeroes, and I believe `removeTrailingZeros` is not actually needed?

Reply:
Yes removing that script and removing `.toFixed(18)` fixed it